source/non_matching/coupling.cc

```diff
@@ -40,6 +40,72 @@
 DEAL_II_NAMESPACE_OPEN
 namespace NonMatching
 {
+  namespace internal
+  {
+    template <int dim0, int dim1, int spacedim>
+    std::pair<std::vector<Point<spacedim>>, std::vector<unsigned int>>
+    compute_used_cells(const GridTools::Cache<dim0, spacedim> &cache,
+                       const DoFHandler<dim1, spacedim> &      immersed_dh,
+                       const Quadrature<dim1> &                quad,
+                       const Mapping<dim1, spacedim> &         immersed_mapping,
+                       const bool                              tria_is_parallel)
+    {
+      const auto &                 immersed_fe = immersed_dh.get_fe();
+      std::vector<Point<spacedim>> used_cell_ids;
+      // Keep track of which cells we actually used
+      std::vector<unsigned int> used_cells;
```

Comment:
and this suggests that it should be a vector of cells, but it's a vector of unsigned ints...